docs/assert/async.md

```diff
@@ -7,17 +7,20 @@ categories:
   - async
 ---
 
-## `async( [ acceptCallCount = 1 ] )`
+## `async( [ acceptCallCount = 1, allowAssertionsAfter = false ] )`
 
 Instruct QUnit to wait for an asynchronous operation.
 
 | name | description |
 |------|-------------|
 | `acceptCallCount` (Number) | Number of expected callbacks before the test is done. Defaults to `1`. |
+| `allowAssertionsAfter` (Boolean) | Whether to allow assertions after the returned callback is invoked. |
```

Comment:
Just throwing this out there for consideration: If we expect more options later, should we just have an options object (which would support just the one key of `allowAssertionsAfter` for now)?